diff --git a/node_modules/shamirs-secret-sharing/random.js b/node_modules/shamirs-secret-sharing/random.js
index 398bf3b..dd87094 100644
--- a/node_modules/shamirs-secret-sharing/random.js
+++ b/node_modules/shamirs-secret-sharing/random.js
@@ -1,7 +1,7 @@
-const { randomBytes } = require('crypto')
+const { getRandomBytes } = require("expo-random");
 
 function random(size) {
-  const r = randomBytes(32 + size)
+  const r = getRandomBytes(32 + size);
   return r.slice(32)
 }
 
